src/Spark.Engine/Service/FhirServiceExtensions/FhirExtensionsBuilder.cs

```diff
@@ -14,11 +14,13 @@ public class FhirExtensionsBuilder : IFhirExtensionsBuilder
         private readonly IStorageBuilder fhirStoreBuilder;
         private readonly Uri baseUri;
         private readonly IList<IFhirServiceExtension> extensions;
+        private readonly SparkSettings sparkSettings;
 
-        public FhirExtensionsBuilder(IStorageBuilder fhirStoreBuilder, Uri baseUri)
+        public FhirExtensionsBuilder(IStorageBuilder fhirStoreBuilder, Uri baseUri, SparkSettings sparkSettings)
```

Comment:
Since it is a public class default SparkSettings to null so we don't break the signature

Reply:
@kennethmyhra done